x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeAutodetectIntegTestCase.java

```diff
@@ -348,17 +349,20 @@ protected void waitForecastToFinish(String jobId, String forecastId) throws Exce
     }
 
     protected ForecastRequestStats getForecastStats(String jobId, String forecastId) {
-        GetResponse getResponse = client().prepareGet()
-                .setIndex(AnomalyDetectorsIndex.jobResultsAliasedName(jobId))
-                .setId(ForecastRequestStats.documentId(jobId, forecastId))
-                .execute().actionGet();
+        SearchResponse searchResponse = client().prepareSearch(AnomalyDetectorsIndex.jobResultsAliasedName(jobId))
+            .setQuery(new BoolQueryBuilder()
+                .filter(new TermQueryBuilder("_id", ForecastRequestStats.documentId(jobId, forecastId))))
```

Comment:
You could use an `IdsQueryBuilder` instead of the `TermQueryBuilder` and drop the `BoolQueryBuilder`

Reply:
@davidkyle TIL :D.